app/controllers/users_controller.rb

```diff
@@ -47,4 +48,8 @@ def user_parameters
                                  :spire,
                                  :staff
   end
+
+  def allow_only_admin
+    deny_access && return unless @current_user.admin?
```

Comment:
Sorry, looks like rubocop  might have changed this. I'll change it back and rerun rubocop to see if it was that.